TrackingTools/TrajectoryState/BuildFile.xml

```diff
@@ -6,11 +6,11 @@
 <use   name="DataFormats/TrackReco"/>
 <use   name="DataFormats/TrajectoryState"/>
 <use   name="DataFormats/BeamSpot"/>
+<use   name="MagneticField/Engine"/>
 <use   name="TrackingTools/AnalyticalJacobians"/>
 <use   name="TrackingTools/TrajectoryParametrization"/>
-<use   name="MagneticField/Engine"/>
 
-<use   name="vdt_headers"/>
+<use   name="vdt"/>
```

Comment:
Giulio, Thanks for the prompt reply. But I need to know which of these two options you prefer:
1) This PR will be closed, and I will submit the changes to these other three files in my PR.s
2) This PR will remain open, with the changes to this one file removed.
Thank you in either case.
Bill